first-analysis-steps/davinci-grid.md

```diff
@@ -37,7 +37,9 @@ j = Job(name='First ganga job')
 myApp = prepareGaudiExec('DaVinci','v44r6', myPath='.')
 j.application = myApp
 j.application.options = ['ntuple_options.py']
-j.application.readInputData('MC_2016_27163002_Beam6500GeV2016MagDownNu1.625nsPythia8_Sim09b_Trig0x6138160F_Reco16_Turbo03_Stripping28NoPrescalingFlagged_ALLSTREAMS.DST.py')
+bkPath = '/MC/2016/Beam6500GeV-2016-MagDown-Nu1.6-25ns-Pythia8/Sim09c/Trig0x6138160F/Reco16/Turbo03/Stripping28r1NoPrescalingFlagged/27163002/ALLSTREAMS.DST'
+data  = BKQuery(bkPath,dqflag=['OK']).getDataset()
```

Comment:
Please add a space after the comma here, `bkPath, dqflag`.

Reply:
Thanks, all done!